### PR TITLE
Added Focus Nodes

### LINKED ISSUE #31: [Feature]Usage of Focus Nodes
**Is your feature request related to a problem? Please describe.**
Currently all the forms in the app do not use focus nodes which means user will have to always select on the next input field manually. Providing an option to navigate to next input field from the keyboard will look great!

**Solution**
Usage of focus nodes.

**Are you working on this?**
Yes.

### PR DESCRIPTION
**Pull request by jaggu21**
Added focus nodes to all the forms.
  
This PR closes #31 . 
Focus Nodes of authentication forms : #34

### CODE REVIEW

**Comment by jaggu21** [comment]
@Nitish145 please let me know if I have missed any form.

**Comment by jaggu21** [comment]
@Nitish145 can you please review this.

File: lib/ui/views/projects/edit_project_view.dart [hunk 1]
```diff
@@ -30,6 +30,14 @@ class _EditProjectViewState extends State<EditProjectView> {
   List<String> _tags;
   final GlobalKey<HtmlEditorState> _descriptionEditor = GlobalKey();
 
+  final _nameFocusNode = FocusNode();
+  final _tagsListFocusNode = FocusNode();
```
Comment (Nitish145) [inline lib/ui/views/projects/edit_project_view.dart:34]: Add empty line before dispose

**Review by Nitish145 — APPROVED** [review]

**Comment by manjotsidhu** [comment]
Can you merge this ? @Nitish145

**Pull request merged**